dor/adapters/converter.py

```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+from uuid import UUID
+
+from cattrs import Converter
+
+
+converter = Converter()
+converter.register_unstructure_hook(datetime, lambda d: d.strftime("%Y-%m-%dT%H:%M:%SZ"))
+converter.register_structure_hook(datetime, lambda d, datetime: datetime.fromisoformat(d))
+
+converter.register_unstructure_hook(UUID, lambda u: str(u))
+converter.register_structure_hook(UUID, lambda u, UUID: UUID(u))
```

Comment:
I put this in `adapters` because my thought was to wrap it with our own class/interface at some point. Do we think that's necessary (yet)? Is adapters the right place for this?

Reply:
I'm okay with `adapters`. I'm not sure I grok what you're thinking --- your `converter.py` feels fine for the scope that this code has?

Reply:
If we wrap the conversion, I think we gain the ability to change what library we use or the implementation generally without changing the code everywhere it's used. But I think this is OK for now, maybe just something to keep in mind as we think more about/deal with more conversion.